hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -227,6 +231,9 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
           <Col xs={24} sm={18} md={12} lg={12} xl={6}>
             <OverviewCard loading={loading} title='Keys' data={keys.toString()} icon='file-text'/>
           </Col>
+          <Col xs={24} sm={18} md={12} lg={12} xl={6}>
+            <OverviewCard loading={loading} title='Pending Key Deletions' data={pendingDeleteKeys.toString()} icon='delete'/>
```

Comment:
This `toString()` raises error for me and @dombizita :

```
TypeError: Cannot read properties of undefined (reading 'toString')
```

<img width="401" alt="image" src="https://user-images.githubusercontent.com/50227127/231851659-5c963620-ce0a-4173-a52f-36cab83ccd97.png">


@smitajoshi12 I believe you will need to check whether `pendingDeleteKeys` is defined or not

Reply:
@smengcl @dombizita 
Checked undefined before toString() function.

Reply:
Thanks @smitajoshi12 .

I'd suggest double checking why the backend is not passing in `pendingDeleteKeys` in the case above. cc @devmadhuu

Reply:
Arafat is working on this PR it will get merged soon.